docs/en/token_endpoint.rst

```diff
@@ -0,0 +1,273 @@
+     "sub":"OP-1234567890",
+     "aud":"https://rp.spid.agid.gov.it/auth",
+     "acr":"https://www.spid.gov.it/SpidL2",
+     "at_hash":"qiyh4XPJGsOZ2MEAyLkfWqeQ",
+     "iat":1519032969,
+     "nbf":1519032969,
+     "exp":1519033149,
+     "jti":"nw4J0zMwRk4kRbQ53G7z",
+     "nonce":"MBzGqyf9QytD28eupyWhSqMj78WNqpc2"
+ }
+
+
+.. list-table:: 
+   :widths: 20 60 20
+   :header-rows: 1
+
+   * - **Claim**
+     - **Description**
+     - **Supported by**
+   * - **iss** 
+     - Identifier of the OP that uniquely indentifies it in the Federation, in Uniform Resource Locator (URL) format. The client must check that this value corresponds to the called OP.
+     - |spid-icon| |cieid-icon|
+   * - **sub** 
+     - For this parameter value, please see the standard "OpenID Connect Core 1.0", "Pairwise Identifier Algorithm". 
+     - |spid-icon| |cieid-icon|
+   * - **aud** 
+     - It contains the client ID. The client must check that this value corresponds to its own client ID.
+     - |spid-icon| |cieid-icon|
+   * - **acr** 
+     - Effective authentication level. It can be equal or greater than the one requested by the client in the Authentication Request.
+     - |spid-icon| |cieid-icon|
+   * - **at_hash** 
+     - Access Token hash. Its value is the base64url coding of the first half of the hash of the value of access_token, using the hashing algorithm indicated in **alg** in the ID Token header. The client must check that this value corresponds to the *Access Token* returned with the ID Token.
+     - |spid-icon| |cieid-icon|
+   * - **iat** 
+     - UNIX Timestamp with the time of JWT generation, coded as NumericDate as indicated in :rfc:`7519`
+     - |spid-icon| |cieid-icon|
+   * - **nbf** 
+     - UNIX Timestamp. Date/time of the validity beginning of the JWT in NumericDate format, as indicated in :rfc:`7519`. It must correspond to the value of **iat**.
+
+       .. code-block:: 
+	   
+	    {
+              userinfo: {...}
+              id_token: {
+                acr: {...},
+                nbf: { essential: true },
+                jti: { essential: true }
+              }
+	    } 
+     - |spid-icon| |cieid-icon|
+   * - **exp**
+     - UNIX Timestamp with the expiry time of the JWT, coded as NumericDate as indicated in :rfc:`7519`
+     - |spid-icon| |cieid-icon|
+   * - **jti** 
+     - String. Unique identifier of the ID Token that the client can use for preventing a reuse, refusing the
+       ID Token if it has already been processed. It must be difficult to guess by an attacker and composed of a casual string. It is suggested to use the *uuid4* generator.
+     - |spid-icon| |cieid-icon|
+   * - **nonce** 
+     - String. Casual string generated by the RP for each user's session and sent in the 
+       Authentication Request (claim nonce), aimed at mitigating replay attacks. The client must make sure that it has the same value as the one tha has been sent in the Authentication Request.
```

Comment:
aimed at -> aimed to or amined for the mitigation of ...

Reply:
aimed regge la preposizione "at"